src/Helper.js

```diff
@@ -75,12 +81,17 @@ const cleanPlanets = async (planetsArray) => {
     const residents = await API.fetchProperty(planet.residents)
     const allResidents = await Promise.all(residents);
     const properPopulation = parseInt(planet.population).toLocaleString('en-US');
+    const imagePath = Image.find(path => {
```

Comment:
one of the testing issues i'm having is this Image.find() still. we may need to get some help